docs/license-handling.asciidoc

```diff
@@ -0,0 +1,63 @@
+[id="{p}-licensing"]
+== Managing licenses in ECK
+
+When you install the default distribution of ECK, you receive a basic license. Any Elastic stack application you manage through
+ECK will also be basic licensed. For the full list of free features that are included in the basic license, see: https://www.elastic.co/subscriptions
+
+[float]
+=== Starting a trial
+If you want to try the features included in the Enterprise subscription, you can start a 30-day trial. To start a trial create
+a Kubernetes secret as shown below:
+
+[source,yaml]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: any-name-you-like
```

Comment:
Let's make this `eck-trial`, I assume most folks will copy and paste this

Reply:
Maybe `eck-enterprise-trial-license` to be even more explicit.